src/server/pfs/server/driver.go

```diff
@@ -3960,6 +3945,19 @@ func (d *driver) listFile(pachClient *client.APIClient, file *pfs.File, full boo
 	})
 }
 
+func validateListFileArgs(file *pfs.File) error {
```

Comment:
This should probably be `validateFile`, especially since you added it to inspect file.